crates/ruff_python_ast/src/token_kind.rs

```diff
@@ -174,7 +174,10 @@ impl TokenKind {
 
     #[inline]
     pub const fn is_unary(&self) -> bool {
-        matches!(self, TokenKind::Plus | TokenKind::Minus | TokenKind::Star)
+        matches!(
```

Comment:
I think it would be good to instead create a new function or move this method to a function because the name unary is now misleading (@ isn't an unary operator)

Reply:
Neither is `TokenKind::Star` in this context, no?

Reply:
That's correct. Blame me 🙈

Reply:
Though I guess `**` is handled as a special-case in the rule (`if kind.is_unary() || kind == TokenKind::DoubleStar`) -- I'll just do that too for `@`.

Reply:
I removed star too .